Directory.Build.props

```diff
@@ -298,8 +298,12 @@
     <!-- Enables Strict mode for Roslyn compiler -->
     <Features>strict;nullablePublicOnly</Features>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <!-- Warnings to always disable -->
-    <NoWarn>$(NoWarn),CS8969</NoWarn>
+    <!-- Warnings to always disable
+         CS8969 -> The private field 'class member' is never used 
+         IDE0031 -> Use null propagation 
+         IDE0074 -> Use compound assignment
+    -->
+    <NoWarn>$(NoWarn);CS8969;IDE0031;IDE0074</NoWarn>
```

Comment:
Why are we suppressing these here and not in the .editorconfig file?

Reply:
also, consider filing issues to reenable? up to you guys though.  but null-propagation and use-compound-assignment are both nice :)